plugins/aws/aws.plugin.zsh

```diff
@@ -13,7 +13,7 @@ function asp() {
   local -a available_profiles
   available_profiles=($(aws_profiles))
   if [[ -z "${available_profiles[(r)$1]}" ]]; then
-    echo "${fg[red]}Profile '$1' not found in '${AWS_CONFIG_FILE:-$HOME/.aws/config}'" >&2
```

Comment:
Why remove the quotes? They help delimit where the name of the profile ends, in case there are spaces in them for example.

Reply:
I'll revert this change.